setup.py

```diff
@@ -7,9 +14,13 @@
     LONG_DESCRIPTION = handle.read()
 
 
+with open('VERSION') as handle:
+    VERSION = handle.read().strip()
```

Comment:
It's important to strip whatever's returned by `read()` as a precaution. Let's say I create a file like so:

    echo 0.18.0 > foo

There's a trailing newline in the file:

```python
>>> with open('foo') as handle:
...     handle.read()
... 
'0.18.0\n'
```